app/models/picture.rb

```diff
@@ -0,0 +1,9 @@
+class Picture < ActiveRecord::Base
+  has_many :comments, dependent: :destroy
+  validates :name, uniqueness: true
+  has_attached_file :image, :styles => {
+    :medium => "300x300>",
+    :thumb => "100x100>" },
+    :default_url => "/images/:style/missing.png"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RegexpLiteral: Use `%r` around regular expression. (https://github.com/bbatsov/ruby-style-guide#percent-r)